workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/static_report.py

```diff
@@ -247,42 +247,53 @@ def diurnal_component_plot(
 
 # Routines for plotting the "diagnostics"
 @timeseries_plot_manager.register
-def rms_plots(diagnostics: Iterable[xr.Dataset]) -> HVPlot:
+def rms_plots(diagnostics: RunDiagnostics) -> HVPlot:
     return plot_1d(diagnostics, varfilter="rms_global")
 
 
 @timeseries_plot_manager.register
-def spatial_mean_plots(diagnostics: Iterable[xr.Dataset]) -> HVPlot:
+def spatial_mean_plots(diagnostics: RunDiagnostics) -> HVPlot:
     return plot_1d_with_region_bar(diagnostics, varfilter="spatial_mean")
 
 
 @timeseries_plot_manager.register
-def spatial_minmax_plots(diagnostics: Iterable[xr.Dataset]) -> HVPlot:
+def spatial_minmax_plots(diagnostics: RunDiagnostics) -> HVPlot:
     return plot_1d_min_max_with_region_bar(
         diagnostics, varfilter_min="spatial_min", varfilter_max="spatial_max"
     )
 
 
 @timeseries_plot_manager.register
-def custom_timeseries_plots(diagnostics: Iterable[xr.Dataset]) -> HVPlot:
+def custom_timeseries_plots(diagnostics: RunDiagnostics) -> HVPlot:
     return plot_1d(diagnostics, varfilter="timeseries")
 
 
 @zonal_mean_plot_manager.register
-def zonal_mean_plots(diagnostics: Iterable[xr.Dataset]) -> HVPlot:
+def zonal_mean_plots(diagnostics: RunDiagnostics) -> HVPlot:
     return plot_1d(diagnostics, varfilter="zonal_and_time_mean")
 
 
 @hovmoller_plot_manager.register
-def zonal_mean_hovmoller_plots(diagnostics: Iterable[xr.Dataset]) -> RawHTML:
+def zonal_mean_hovmoller_plots(diagnostics: RunDiagnostics) -> RawHTML:
     return plot_2d_matplotlib(diagnostics, "zonal_mean_value", ["time", "latitude"])
 
 
 @hovmoller_plot_manager.register
-def zonal_mean_hovmoller_bias_plots(diagnostics: Iterable[xr.Dataset]) -> RawHTML:
+def zonal_mean_hovmoller_bias_plots(diagnostics: RunDiagnostics) -> RawHTML:
     return plot_2d_matplotlib(diagnostics, "zonal_mean_bias", ["time", "latitude"])
 
 
+@hovmoller_plot_manager.register
+def zonal_mean_short_term_bias_plots(diagnostics: RunDiagnostics) -> RawHTML:
+    return plot_2d_matplotlib(
+        diagnostics.trim_duration(np.timedelta64(3, "D")),
+        "zonal_mean_bias",
+        ["time", "latitude"],
+        title="3-day zonal mean bias",
+        plot_vars=[v.lower() for v in GLOBAL_AVERAGE_DYCORE_VARS],
```

Comment:
It's unfortunate we have to do this `.lower()` here (and in other places) to deal with the fact that for some reason [we do a lower-case transformation](https://github.com/ai2cm/fv3net/blob/12d2ed3c259ea669d7e21f29133c7e3d83ca054d/workflows/diagnostics/fv3net/diagnostics/prognostic_run/compute.py#L74) when saving diagnostics. At some point we should stop doing that, but will be tricky to be backwards compatible.